fact-bounty-client/src/pages/ForgotPassword/ForgotPassword.jsx

```diff
@@ -0,0 +1,250 @@
+            {errors.email}
+          </Typography>
+        </FormControl>
+      )
+      buttonName = 'Send verification code'
+    } else {
+      formInput = (
+        <FormControl margin="normal" required fullWidth>
+          <InputLabel htmlFor="verificationToken">
+            Verification token
+          </InputLabel>
+          <Input
+            autoComplete="on"
+            onChange={this.onChange}
+            value={this.state.verificationToken}
+            id="verificationToken"
+          />
+        </FormControl>
+      )
+      buttonName = 'Verify token'
+    }
+    return (
+      <main className={this.props.classes.main}>
+        <CssBaseline />
+        <Paper className={this.props.classes.paper}>
+          {errors.fetch ? (
+            <Toast
+              open={openToast}
+              onClose={this.closeToast}
+              message="Something went wrong, Try again later"
+              variant="error"
+            />
+          ) : null}
+          <Avatar className={this.props.classes.avatar}>
+            <LockOutlinedIcon />
+          </Avatar>
+          <Typography component="h1" variant="h5">
+            Forgot Password
+          </Typography>
+
+          <form
+            noValidate
+            onSubmit={this.onSubmit}
+            className={this.props.classes.form}
+          >
+            <Typography component="span" variant="caption" color="error">
+              {typeof this.props.errors === 'object'
+                ? this.props.errors.message
+                : null}
+            </Typography>
+            <Typography
+              component="span"
+              variant="caption"
+              color="primary"
+              align="center"
+            >
+              {typeof this.props.success === 'object'
+                ? this.props.success.message
+                : null}
+            </Typography>
+            {formInput}
+            <Button
+              type="submit"
+              fullWidth
+              variant="contained"
+              color="primary"
+              className={this.props.classes.submit}
+              disabled={!this.state.formValid}
+            >
+              {buttonName}
+            </Button>
+          </form>
+        </Paper>
+      </main>
+    )
+  }
+}
+
+ForgotPassword.propTypes = {
+  forgotPassword: PropTypes.func.isRequired,
+  authVerificationToken: PropTypes.func.isRequired,
+  auth: PropTypes.object.isRequired,
+  errors: PropTypes.object.isRequired,
+  success: PropTypes.object.isRequired,
+  history: PropTypes.object,
+  classes: PropTypes.object,
+  updateError: PropTypes.func.isRequired,
+  updateSuccess: PropTypes.func.isRequired
+}
+
+const mapStateToProps = state => ({
+  auth: state.auth,
+  errors: state.errors,
+  success: state.success
+})
+
+export default compose(
+  withStyles(styles, {
+    name: 'ForgotPassword'
+  }),
+  connect(mapStateToProps, { forgotPassword, authVerificationToken, updateError, updateSuccess })
```

Comment:
Codacy build is failing on this line because of linelength, try breaking it in multiple line.

Reply:
i tried every possible way to refactor this line, but it keeps on failing. But if you say i will try it again.